lib/ansible/modules/storage/netapp/netapp_e_lun_mapping.py

```diff
@@ -89,7 +103,9 @@ def __init__(self):
         argument_spec.update(dict(
             state=dict(required=True, choices=["present", "absent"]),
             target=dict(required=False, default=None),
-            volume_name=dict(required=True)))
+            volume_name=dict(required=True, aliases=["volume"]),
+            lun=dict(required=False),
```

Comment:
Why has this been added?

Reply:
We simply found volume_name to be unnecessarily verbose and thought adding volume as an alias would help.